src/engine/sync/synccontrol.cpp

```diff
@@ -519,8 +519,6 @@ void SyncControl::slotSyncLeaderEnabledChangeRequest(double state) {
         // Turning off leader goes back to follower mode.
         switch (mode) {
         case SyncMode::LeaderExplicit:
-            m_pChannel->getEngineBuffer()->requestSyncMode(SyncMode::LeaderSoft);
```

Comment:
yes this looks right

Reply:
Wasn't this intentional? I think the explicit leader also enabled scratching both decks in sync. In case you don't want this but you're unhappy with Mixxx automatic selection, the trick was to make a deck explicit leader, then disable the explicit leader to make it soft leader.

Reply:
hm good question

Reply:
yeah that could be confusing to users, it might pick a different  deck.  However in practice, when implicit leader is active there's no functional difference between decks.  Adjusting rate on any deck will adjust rates on all other decks.

I think I agree, we want to keep this the way it was and just focus on the specific bug